world_universities_and_domains.json

```diff
@@ -107818,18 +107818,6 @@
       ],
       "country": "United States"
   },
-  {
-      "web_pages": [
-          "http://www.dupage.edu"
-      ],
-      "name": "College of DuPage",
-      "alpha_two_code": "US",
-      "state-province": null,
-      "domains": [
-          "dupage.edu"
-      ],
-      "country": "United States"
-  },
```

Comment:
Expand down the changed file and you will find the next record with a working website that is http://www.cod.edu/.